iamproperty/iamproperty.github.io
# Documentation to support illustrations with an example.

## Summary of Changes:
1. Create the documentation around the illustrations and demo how they can be used.
2. Add in a few of the SVG assets for the characters

## Rationale
Demo how the illustration SVGs can be used.

## How to Test
Open up the preview site (see the below netlify comment below) and navigate to /foundations/illustrations

## Ticket(s)
DS-32

## Checklist

## Notes
This is just a demo page
